src/auto-evo/selection_pressure/RootPressure.cs

```diff
@@ -0,0 +1,41 @@
+﻿namespace AutoEvo;
+
+using System;
+
+public class RootPressure : SelectionPressure
+{
+    // Needed for translation extraction
+    // ReSharper disable ArrangeObjectCreationWhenTypeEvident
+    public static readonly LocalizedString Name = new LocalizedString("ROOT_PRESSURE");
```

Comment:
I think some of the new auto-evo translation keys are too simple for translators to understand. Maybe prefixing most if not all of them with `MICHE_` or here with `PRESSURE_` ? Prefixes also help the localization tool from grouping the related text to be translated at once.